Source/Core/Simon1994PlanetaryPositions.js

```diff
@@ -80,27 +80,33 @@ define([
             inclination = -inclination;
             longitudeOfNode += CesiumMath.PI;
         }
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
Took me a moment of looking hard at the diff before I saw your second comment :smile: 
I'll add it, it's nice to have a bit of separation between the pragma blocks and other code.